scipy/optimize/_qap.py

```diff
@@ -0,0 +1,116 @@
+import numpy as np
+from . import linear_sum_assignment, OptimizeResult
+
+
+def umeyama_eigendecomposition(A, B, maximize=False):
+    """Finds an approximate solution to QAP using Umeyama's method."""
+    # 'Correct' non-hermitian matrices
+    if (A != A.T).any() or (B != B.T).any():
+        A = (A + A.T) / 2 + 1j * (A - A.T) / 2
+        B = (B + B.T) / 2 + 1j * (B - B.T) / 2
+
+    va = np.linalg.eigh(A)[1]
+    vb = np.linalg.eigh(B)[1]
+    W = np.abs(vb) @ np.abs(va).T
```

Comment:
Do you mean va.real and vb.real here? Why abs?

Reply:
Umeyama's paper takes the magnitude:
![image](https://user-images.githubusercontent.com/6570539/89695311-6226f300-d8c8-11ea-9b67-209da6bf3bc2.png)

Reply:
That is my understanding of it too.
Going to mark as resolved.